server/src/main/java/org/gluu/oxtrust/api/client/RegistrationManagementResponse.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * oxTrust is available under the MIT License (2008). See http://opensource.org/licenses/MIT for full text.
+ *
+ * Copyright (c) 2014, Gluu
+ */
+package org.gluu.oxtrust.api.client;
+
+import org.codehaus.jackson.map.DeserializationConfig;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.xdi.config.oxtrust.AppConfiguration;
+import org.xdi.model.GluuAttribute;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Wrapper for registration management service's response.
+ *
+ * @author Shoeb Khan
+ * @version July 06, 2018
+ */
+public class RegistrationManagementResponse {
+
+
+    private List<GluuAttribute> selectedAttributes;
+
+    private List<GluuAttribute> attributes;
+
+    private boolean captchaDisabled;
+
+    private AppConfiguration oxTrustappConfiguration;
```

Comment:
We shouldn't expose this field as it is, it contains sensitive data:
KeystorePassword
idpSecurityKeyPassword
OxAuthClientPassword
...